source/declarations.tex

```diff
@@ -1780,19 +1824,17 @@
 \end{example}
 
 \pnum
-Redeclarations or specializations of a function or function template with a
-declared return type that uses a placeholder type shall also use that
-placeholder, not a deduced type.
-Similarly,
-redeclarations or specializations of a function or function template with a
-declared return type that does not use a placeholder type
-shall not use a placeholder.
+If a function or function template has
+a declared return type that uses a placeholder type,
+redeclarations or specializations (respectively) of it shall use that
+placeholder, not a deduced type;
+otherwise, they shall not use a placeholder.
```

Comment:
The problem with the reformulation is that "R" is only in scope if the preceding "If .. has a declared return type that uses a placeholder type" is true. But the "otherwise" is talking about all the other cases.
So, I'm leaving "they" in (the only plural in the sentence is "redeclarations or specializations", so we're pretty unambiguous), pending an out-of-line rewrite if desired. The rest is fixed.